src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,50 @@
-export const Pagination = () => {};
+import React from "react";
+import { getNumbers } from "../../utils";
+
+interface Props {
+  buttonsLength: number,
+  activePage: number,
+  callback: (page:number) => void,
+}
+
+export const Pagination : React.FC<Props> = ({buttonsLength, activePage, callback}) => {
+  const buttonsItems = getNumbers(1, buttonsLength);
+
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${activePage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
```

Comment:
Using `href="#prev"` for the previous link might not be ideal for accessibility. Consider using a `button` element instead of an `a` tag for actions that don't navigate to a new page.